crest/Assets/Crest/Crest/Scripts/LodData/RegisterHeightInput.cs

```diff
@@ -83,7 +83,8 @@ public override bool Validate(OceanRenderer ocean, ValidatedHelper.ShowMessage s
 
             if (isValid)
             {
-                if (ocean != null && ocean._simSettingsAnimatedWaves._renderTextureGraphicsFormat != GraphicsFormat.R32G32B32A32_SFloat)
+                if (ocean != null && ocean._simSettingsAnimatedWaves != null &&
```

Comment:
This could be removed or adapted for depth if you think it is a problem still.

Reply:
oh nice, removed